brave/browser-laptop
# add a cookie manager

this allows users to view all the cookies, sort them, and delete individual ones or in bulk.

fix https://github.com/brave/browser-laptop/issues/1991

todo: add ability to edit cookies and add to global whitelist/blacklist

test plan:
1. automated tests should pass
2. browse some sites in various browsing sessions (session tab, regular tab, private tab, etc.)
3. go to about:cookies
4. you should see cookies for those sites
5. click on a cookie and right-click to delete or copy. both should work.
6. repeat step 5 but for multiple selected cookies at once
7. select some cookies, hit delete key. they should be deleted.

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
